build-conflicts.php

```diff
@@ -109,32 +111,34 @@ static function ($errorCode, $message = '', $file = '', $line = 0) : bool {
         return $output;
     };
 
-    $cleanBuildDir = static function () use ($buildDir, $execute) : void {
-        $execute('rm -rf ' . escapeshellarg($buildDir));
-        $execute('mkdir ' . escapeshellarg($buildDir));
-    };
-
-    $cloneAdvisories = static function () use ($advisoriesRepository, $buildDir, $execute) : void {
-        $execute(
-            'git clone '
-            . escapeshellarg($advisoriesRepository)
-            . ' ' . escapeshellarg($buildDir . '/security-advisories')
+    $cleanBuildDir = static fn () : array =>
+        array_map(
+            $execute,
+            [
+                $argsToString('rm -rf', $buildDir),
+                $argsToString('mkdir', $buildDir),
+            ]
         );
-    };
 
-    $cloneRoaveAdvisories = static function () use ($roaveAdvisoriesRepository, $buildDir, $execute) : void {
-        $execute(
-            'git clone '
-            . escapeshellarg($roaveAdvisoriesRepository)
-            . ' ' . escapeshellarg($buildDir . '/roave-security-advisories')
-        );
+    $cloneAdvisories = static fn () : array  =>
+        $execute($argsToString('git clone', $advisoriesRepository, $buildDir . '/security-advisories'));
 
-        $execute(sprintf(
-            'cp -r %s %s',
-            escapeshellarg($buildDir . '/roave-security-advisories'),
-            escapeshellarg($buildDir . '/roave-security-advisories-original')
-        ));
-    };
+    $cloneRoaveAdvisories = static fn () : array =>
```

Comment:
I tried to use new "arrow function" syntax in order to drop the `use()` statements, but I'm not really sure it makes sense now. File has 42 new lines and 40 lines removed. Should I revert it ?

Reply:
I don't mind either way: just need to see if all the tooling still understands arrow functions...